group_vars/maas_region_controller/10-maas_region_controller

```diff
@@ -7,9 +7,7 @@ maas_https_port: 5443       # The https port MAAS uses to communicate on
 admin_username: "admin"
 admin_password: "admin"
 admin_email: "admin@email.com"
-admin_id: "admin"
-
-import_ssh_keys: true
+admin_id: ""
```

Comment:
This can just be removed if there is the conditional check with is defined below. Otherwise, if default is blank, then the conditional check should not even be necessary I think- since it will by default be defined (as blank) and I think it prioritises vars by the user